src/test/java/core/basesyntax/MyHashMapTest.java

```diff
@@ -2,8 +2,10 @@
 
 import org.junit.Assert;
 import org.junit.Test;
+
 import java.lang.reflect.Array;
```

Comment:
Unused import statement. Remove the `java.lang.reflect.Array` import as it is not used in the tests.